vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -3059,8 +3059,7 @@ quantitykind:CorrelatedColorTemperature
   qudt:informativeReference "https://www.lrc.rpi.edu/programs/nlpip/lightinganswers/lightsources/whatiscct.asp#:~:text=Correlated%20color%20temperature%20(CCT)%20is,required%20to%20specify%20a%20chromaticity."^^xsd:anyURI ;
   qudt:plainTextDescription "Correlated color temperature (CCT) is a measure of light source color appearance defined by the proximity of the light source's chromaticity coordinates to the blackbody locus, as a single number rather than the two required to specify a chromaticity." ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
-  rdfs:label "Correlated Color Temperature"@en-us ;
-  rdfs:label "Correlated Colour Temperature"@en ;
+  rdfs:label "Correlated Colour Temperature"@en ; , "Correlated Color Temperature"@en-us
```

Comment:
Please delete the ; and put it at the end.

Reply:
Also, please merge the main branch into your branch, because GitHub is reporting a merge conflict.